src/jdk.crypto.ec/share/classes/sun/security/ec/ECDHKeyAgreement.java

```diff
@@ -160,23 +161,37 @@ private static void validateCoordinate(BigInteger c, BigInteger mod)
         }
     }
 
-    /*
-     * Check whether a public key is valid.
-     */
+    // Check whether a public key is valid, following the ECC
```

Comment:
Coding conventions used to be /* */ for method description and multiline comments.

Otherwise, this work is much appreciated for any maintainers.